improver_tests/calibration/reliability_calibration/test_ManipulateReliabilityTable.py

```diff
@@ -186,6 +186,26 @@ def test_non_monotonic(self):
         )
         assert_array_equal(result.data, expected_result)
 
+    def test_non_monotonic_lower_forecast_count_on_left(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6088429)